application/src/main/java/com/sanction/lightning/resources/FacebookResource.java

```diff
@@ -128,15 +138,22 @@ public Response publish(@Auth Key key, @QueryParam("username") String username,
                                @FormDataParam("file") InputStream inputStream,
                                @FormDataParam("file") FormDataContentDisposition
                                          contentDispositionHeader,
-                               String message) {
+                               @QueryParam("type") String type,
+                               @FormDataParam("message") @DefaultValue("") String message,
+                               @FormDataParam("title") @DefaultValue("") String videoTitle) {
```

Comment:
Can you align these white-space wise with the `@Auth` tag? That way the `@FormDataParam("file") FormDataContentDisposition contentDispositionHeader` can all fit on one like hopefully.